From b38a92d07e5f87733a4aee8941f2f3945bed8334 Mon Sep 17 00:00:00 2001
From: Tin Tran <tin.tran.xk@renesas.com>
Date: Fri, 30 May 2025 15:35:50 +0700
Subject: [PATCH 46/46] PCIe: dwc: rcar: Temporarily remove PERST# handling
 from driver

Since I2C0 having conflict resource issue, get PERST#
signal through I2C0 will result in PCIe4 being probe pending.

Signed-off-by: Tin Tran <tin.tran.xk@renesas.com>
---
 drivers/pci/controller/dwc/pcie4-rcar-host.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pci/controller/dwc/pcie4-rcar-host.c b/drivers/pci/controller/dwc/pcie4-rcar-host.c
index b8fa6814aa3e..0978f88f46f1 100644
--- a/drivers/pci/controller/dwc/pcie4-rcar-host.c
+++ b/drivers/pci/controller/dwc/pcie4-rcar-host.c
@@ -421,7 +421,7 @@ int rcar_gen5_pcie_devm_reset_get(struct rcar_pcie4 *rcar_pcie4,
 	rcar_pcie4->perst = devm_gpiod_get(dev, "reset", GPIOD_OUT_LOW);
 	if (IS_ERR(rcar_pcie4->perst)) {
 		dev_err(dev, "Failed to get PERST GPIO\n");
-		return PTR_ERR(rcar_pcie4->perst);
+		//return PTR_ERR(rcar_pcie4->perst);
 	}
 
 	return 0;
-- 
2.34.1

